Dockerfile-OpenMPI

```diff
@@ -96,20 +96,25 @@ COPY Optimization ${HOME}/Lake_Problem_DPS/Optimization
 COPY Re-evaluation ${HOME}/Lake_Problem_DPS/Re-evaluation
 COPY run_lake_problem.sh ${HOME}/Lake_Problem_DPS
 
-COPY Optimization/moeaframework.h ${HOME}/Lake_Problem_DPS/Optimization/borg
-COPY Optimization/moeaframework.c ${HOME}/Lake_Problem_DPS/Optimization/borg
+COPY Optimization/moeaframework.h ${HOME}/Lake_Problem_DPS/Optimization/borg/
+COPY Optimization/moeaframework.c ${HOME}/Lake_Problem_DPS/Optimization/borg/
 
 RUN chown -R ${nixuser}:${nixuser} ${HOME}/Lake_Problem_DPS/DataInPaper
 RUN chown -R ${nixuser}:${nixuser} ${HOME}/Lake_Problem_DPS/FigureGeneration
 RUN chown -R ${nixuser}:${nixuser} ${HOME}/Lake_Problem_DPS/Optimization
 RUN chown -R ${nixuser}:${nixuser} ${HOME}/Lake_Problem_DPS/Re-evaluation
 
-
-# TODO: remove borg before making public.  Temporary for testing.
-
 USER $nixuser
+
+ARG SSH_PRIVATE_KEY
 COPY Docker/default.nix $ENVSDIR/
-RUN $nixenv && cd /tmp && nix-shell $ENVSDIR/default.nix --run ${HOME}/Lake_Problem_DPS/Optimization/DPS/compile_DPS.sh
+#Suggest multi-stage build to make sure key not visible
```

Comment:
This comment should have `TODO: ` at the beginning so we know it's a remaining item to do, not just a comment.